### PR TITLE
fixed segfault regression

### PR DESCRIPTION
**Pull request by scopatz**
This should allow signals to be reported properly, again.  Mea culpa.

### CODE REVIEW

**Comment by scopatz** [comment]
ping!

**Comment by astronouth7303** [comment]
Nothing horribly stands out.

**Comment by scopatz** [comment]
@astronouth7303 please feel free to merge!

File: xonsh/proc.py [hunk 1]
```diff
@@ -872,6 +872,16 @@ def returncode(self):
...
         return self.proc.returncode
 
+    @property
+    def signal(self):
+        """Process signal, or None."""
+        s = getattr(self.proc, "signal", None)
+        if s is None:
+            rtn = self.returncode
+            if rtn is not None and rtn != 0:
+                s = (-1*rtn, os.WCOREDUMP(rtn))
```
Comment (astronouth7303) [inline xonsh/proc.py]: `WCOREDUMP` is not available on Windows.

**Comment by astronouth7303** [comment]
Sorry, nevermind. Spotted a problem.

File: xonsh/proc.py (hunk 1 above)
Comment (scopatz) [inline xonsh/proc.py]: Ok,  believe I have a fix for this now.

**Comment by gforsyth** [comment]
Thanks, @scopatz !

**Pull request merged**